include/exec/sequence_senders.hpp

```diff
@@ -449,52 +545,73 @@ namespace exec {
       && sender_to<next_sender_of_t<_Receiver, _Sender>, __stopped_means_break_t<_Receiver>>;
 
     template <class _Sequence, class _Receiver>
-    concept __subscribable_with_static_member =
-      receiver<_Receiver> && sequence_sender_in<_Sequence, env_of_t<_Receiver>>
-      && sequence_receiver_from<_Receiver, _Sequence>
-      && requires(_Sequence&& __sequence, _Receiver&& __rcvr) {
-           {
-             STDEXEC_REMOVE_REFERENCE(_Sequence)
-             ::subscribe(static_cast<_Sequence&&>(__sequence), static_cast<_Receiver&&>(__rcvr))
-           };
-         };
+    using __subscribe_member_result_t = decltype(__declval<_Sequence>()
+                                                   .subscribe(__declval<_Receiver>()));
```

Comment:
@kirkshoop could you pls give this a spot check? i had to reconstruct some of your work after a bad rebase/force push, and i'm not 100% sure i got everything back the way it should be.

Reply:
I checked, it is close enough. Ship it! - None of the remaining differences are functional, just spelling and strings..